src/Computer.js

```diff
@@ -0,0 +1,43 @@
+const { Random } = require('@woowacourse/mission-utils');
+const ValidationError = require('./ValidationError');
+
+class Computer {
+  static createUniqueNumbers(start, end, count) {
+    Computer.#validateUniqueNumbers(start, end, count);
+
+    const numberSet = new Set();
+
+    while (numberSet.size !== count) {
+      const number = Random.pickNumberInRange(start, end);
+      if (!numberSet.has(number)) numberSet.add(number);
+    }
```

Comment:
Set을 이용한 방법도 좋네요! 
심지어 set을 이용하면, has로 평가하지 않고, 크기로만 평가해도 될것 같은걸요?:) 
```suggestion
    const numberSet = new Set();

    while (numberSet.size !== count) {
      const number = Random.pickNumberInRange(start, end);
      numberSet.add(number);
    }
```

Reply:
조건을 불필요하게 확인하고 있었네요! 감사합니다 :)